firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/DialogUtils.java

```diff
@@ -0,0 +1,78 @@
+package com.google.firebase.app.distribution;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.Context;
+import android.content.DialogInterface.OnCancelListener;
+import android.content.DialogInterface.OnClickListener;
+import androidx.annotation.NonNull;
+import com.google.firebase.FirebaseApp;
+
+class DialogUtils {
+
+  private DialogUtils() {}
+
+  static AlertDialog getSignInDialog(
```

Comment:
These methods don't really need to return the dialog since all the caller does in each case is call `show()` on it. For convenience you might as well just change these to `void showSomeDialog()` methods.